pkg/resources/util.go

```diff
@@ -154,3 +155,11 @@ func ReconcileRedis(ctx context.Context, client client.Client, deploymentType, t
 
 	return r, nil
 }
+
+// checks to see if env var exists
+func EnvOrDefault(key string, fallback string) string {
```

Comment:
I think we can remove this util function now.

Reply:
already done